x/leverage/README.md

```diff
@@ -1,98 +1,132 @@
-# Overview
+# Leverage Module
 
-This document covers basic concepts and math that determine the `leverage` module's behavior.
+## Abstract
 
-## Accepted Assets
+This document specifies the `x/leverage` module of the Umee chain.
 
-At the foundation of the `leverage` module is the [Token Registry](02_state.md#Token-Registry), which contains a list of accepted types.
+The leverage module allows users to supply and borrow assets, and implements various features to support this, such as a token accept-list, a dynamic interest rate module, incentivized liquidation of undercollateralized debt, and automatic reserve-based repayment of bad debt.
+
+The leverage module depends directly on `x/oracle` for asset prices, and interacts indirectly with `x/ibctransfer`, `x/peggy`, and the cosmos `x/bank` module as these all affect account balances.
+
+## Contents
+
+1. **[Concepts](#concepts)**
+   - [Accepted Assets](#Accepted-Assets)
+     - [uTokens](#uTokens)
+   - [Supplying and Borrowing](#Supplying-and-Borrowing)
+   - [Reserves](#Reserves)
+   - [Liquidation](#Liquidation)
+   - Important Derived Values:
+     - [Adjusted Borrow Amounts](#Adjusted-Borrow-Amounts)
+     - [uToken Exchange Rate](#uToken-Exchange-Rate)
+     - [Supply Utilization](#Supply-Utilization)
```

Comment:
Lint is erroring here but this link works fine

Reply:
I think it complains about capital letters. Going to check it.

Reply:
yeap, that was it!